www/docs/client/links/httpBatchLink.md

```diff
@@ -0,0 +1,147 @@
+---
+id: http-batch
+title: HTTP Batch Link
+sidebar_label: HTTP Batch Link
+slug: /links/http-batch
+---
+
+`httpBatchLink` is a [**terminating link**](./overview#the-terminating-link) that batches an array of individual tRPC operations into a single HTTP request that's sent to a single tRPC procedure. tRPC uses `httpBatchLink` by default.
```

Comment:
Not default anymore

Reply:
really? I checked the vanilla client builder in v10 it still uses the httpBatchLink when no links are passed. Check here: https://github.com/trpc/trpc/blob/next/packages/client/src/createTRPCClient.ts#L18

Reply:
Note the ` * @deprecated use `createTRPCProxyClient` instead`

Reply:
So how does it work now? It seems that, according to [this,](https://github.com/trpc/trpc/blob/next/packages/client/src/internals/TRPCClient.ts#L81) the user needs to manually specify the terminating link they are going to use in v10 even if they don't use any other links?

Reply:
well, after looking at some of the example projects, it seems it works like I described above. However, that leaves me with a little confusion, is request batching still enabled by default in `@trpc/server`? And if so, what is the behavior when using `httpLink` and not `httpBatchLink` by having `batching` enabled in `@trpc/server`?